### ISSUE TITLE
Documentation lacking

### ISSUE THREAD

**Issue by UnitedMarsupials-zz** [sensu-plugins/sensu-plugins-graphite#34]
After adding the `graphite`-handler as per the README:

``` json
    "handlers": {
        "default": {
            "type": "set",
            "handlers": [
                "stdout"
            ]
        },
        "stdout": {
            "type": "pipe",
            "command":  "cat"
        },
        "graphite": {
            "server":   "IP.ADD.RE.SS",
            "port": 2003
        }
    }
```

I now have sensu-client refusing to start:

```
"invalid settings"
"handler type must be a string","object":{"server":"172.30.77.64","port":2003,"name":"graphite"}
"unknown handler type","object":{"server":"172.30.77.64","port":2003,"name":"graphite"}
"SENSU NOT RUNNING!"
```

I'm sure, I'll figure it out eventually, but the documentation should offer better examples.

**Comment by scosist** [comment]
Agreed, more examples would help. I got it working with the following:

```
{
  "handlers":
  {
    "graphite": {
       "type": "tcp",
       "socket": {
         "host":"graphite",
         "port":2003
       },
       "mutator": "only_check_output"
    }
  }
}
```

**Comment by majormoses** [comment]
@scosist any chance I could get you to enhance some documentation with this?

**Comment by scosist** [comment]
Sure, would that involve a PR against the README.md with an example and some notes or is there another place where more documentation exists for this?

**Comment by majormoses** [comment]
just readme or in the check headers currently. Given that it affects multiple checks readme is more appropriate.

**Comment by majormoses** [comment]
@UnitedMarsupials @scosist I think this can be closed with #54 if you feel otherwise just comment and I will re-open.
